more_itertools/more.py

```diff
@@ -33,6 +33,7 @@
     powerset,
     take,
     unique_everseen,
+    prepend,
```

Comment:
Hmm. @bbayles, if I remove the `prepend` then Doctest fails, if I add `prepend` then flake8 fails. What now?

Reply:
It shouldn't be imported here; it should be imported in the docstring before [this line](https://github.com/more-itertools/more-itertools/pull/475/files#diff-d7dff201bd69500a573b7e96c52f8e625242d769a04621224112faf6e7bc63fdR3576).

Reply:
Duh. Commit https://github.com/more-itertools/more-itertools/pull/475/commits/254133ff846af44c99150b645bd73e4999bd06a3.